composer.json

```diff
@@ -7,21 +7,21 @@
     ],
     "license": "GPL-3.0",
     "require": {
-        "php": "^8.1|^8.2",
+        "php": "^8.2",
         "guzzlehttp/guzzle": "^7.2",
-        "laravel/framework": "^10.10",
-        "laravel/tinker": "^2.8",
-        "rapidez/account": "~0.16",
-        "rapidez/core": "^1.7",
-        "rapidez/image-resizer": "~0.14",
-        "rapidez/menu": "~0.4"
+        "laravel/framework": "^11.0",
+        "laravel/tinker": "^2.9",
+        "rapidez/account": "^0",
```

Comment:
`^0` is a special case in semver where it means the same as `^1.0`
https://jubianchi.github.io/semver-check/#/^0/0.5
https://jubianchi.github.io/semver-check/#/^0/1.0
whereas `^0.0` only allows patch versions instead of minor.
https://jubianchi.github.io/semver-check/#/^0.0/0.1